packages/contracts-bedrock/scripts/libraries/ForgeArtifacts.sol

```diff
@@ -190,6 +191,19 @@ library ForgeArtifacts {
     function getInitializedSlot(string memory _contractName) internal returns (StorageSlot memory slot_) {
         string memory storageLayout = getStorageLayout(_contractName);
 
+        // Contracts that use oz v5 should be here
+        if (LibString.eq(_contractName, "L1ERC721Bridge")) {
```

Comment:
Do more contracts use oz v5 now? if so then we can know the slot always and just only return this value

Reply:
Unfortunately not, I made a timeboxed effort to get #12356 merged so that I could build on top of it, but ran into surprising e2e failures that I didn't have time to debug.